spark/src/main/kotlin/com/adevinta/spark/components/bottomsheet/BottomSheet.md

```diff
@@ -0,0 +1,51 @@
+# Package com.adevinta.spark.components.bottomsheet
+
+[BottomSheet](https://spark.adevinta.com/1186e1705/p/67d41e-bottom-sheet/b/02056b)
+A bottom sheet is a UI component commonly used in mobile applications to present additional content
+or options from the bottom of the screen.
+It is a modal component that slides up from the bottom of the screen and covers the entire screen.
+
+#### BottomSheet
+
+```kotlin
+fun BottomSheet(
```

Comment:
This is already present in Dokka, we should present how to use the component here